src/components/Blog.js

```diff
@@ -0,0 +1,212 @@
+import React, { useEffect, useState } from 'react';
+import { hot } from 'react-hot-loader';
+import PropTypes from 'prop-types';
+import {
+  Grid,
+  Typography,
+  Card,
+  Box,
+  Link,
+  Paper,
+  Button,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import QuestionRender from './QuestionRender';
+
+const useStyles = makeStyles({
+  cardcontent: {
+    '&:last-child': { paddingBottom: 16 },
+  },
+  markdown: {
+    '& p': { marginBottom: 0, marginTop: 0, overflowWrap: 'anywhere' },
+  },
+  white: {
+    '&.cardroot': {
+      backgroundColor: 'white',
+    },
+    '&.noAnswer': {
+      opacity: '70%',
+      fontStyle: 'italic',
+    },
+  },
+  gray: {
+    '&.cardroot': {
+      backgroundColor: '#F9F9F9',
+    },
+    '&.noAnswer': {
+      opacity: '70%',
+      fontStyle: 'italic',
+    },
+  },
+  unfinishedRequired: {
+    color: 'red',
+    '&.cardroot': {
+      borderColor: 'red',
+    },
+  },
+});
+
+const Blog = ({
+  questions, saveBlogAnswers, isReadOnly, programID,
+}) => {
+  const [editMode, setEditMode] = useState(false);
+  const [blogQuestions, setBlogQuestions] = useState([]);
+  const classes = useStyles();
+
+  useEffect(() => {
+    setBlogQuestions(questions);
+  }, [questions]);
+
+  const toggleEditMode = () => {
+    if (editMode) {
+      saveBlogAnswers(programID, blogQuestions);
+    }
+    setEditMode(!editMode);
+  };
+
+  const getClass = () => {
+    if (!isReadOnly && editMode) {
+      return classes.gray;
+    }
+    return classes.white;
+  };
+
+  const getRequiredClass = (required, answer) => {
+    if (required && !answer && !isReadOnly) {
+      return classes.unfinishedRequired;
+    }
+    return null;
+  };
+
+  return (
+    <Grid container direction="column">
+      {questions.length === 0 && isReadOnly ? (
```

Comment:
Could all of these where `{` is on the same line as the javascript be formatted like below to be consistent with the rest of the application?
```
{
  <javascript> ? (
  ) : (
  )
}
```
Basically, have the `{` and `}` on their own lines with the rest indented within it